tools/firmware_hash.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+from hashlib import blake2s
+from os import sys
+
+FILE_T1 = "legacy/firmware/trezor.bin"
+FILE_T2 = "core/build/firmware/firmware.bin"
+
+SIZE_T1 = (7 * 128 + 64) * 1024
+SIZE_T2 = 13 * 128 * 1024
+
+for filename, size in ((FILE_T1, SIZE_T1), (FILE_T2, SIZE_T2)):
+    try:
+        data = open(filename, "rb").read()
+    except:
```

Comment:
`except FileNotFoundError` would be more descriptive

Reply:
[d429101](https://github.com/trezor/trezor-firmware/pull/2239/commits/d429101cb44669cedb377199a7e5939391e58594)